docs/flashbots-protect/rpc/large-transactions.md

```diff
@@ -0,0 +1,10 @@
+---
+title: large transaction allowlist
+---
+
+Some projects need to post transactions that are larger in size (e.g. measured by bytes) than the txpool traditionally allows. Flashbots bundles are not subject to the same constraints. However, at the RPC level only transactions to certain contracts are allowed to be over 128kb.
```

Comment:
```suggestion
Some projects need to post transactions that are larger in size (measured in bytes) than the txpool traditionally allows. Flashbots bundles are not subject to the same constraints. However, at the RPC level only transactions to certain contracts are allowed to be over 128kb.
```

This would be i.e. instead of e.g. I suggest to just remove the acronym.